UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -221,52 +224,67 @@ public void AffectDamage(float damage, int damageType)
 			// May also want it so it can miss sub organs
 			if (DamageSubOrgans && OrganList.Count > 0)
 			{
-				var organDamageRatingValue = SubOrganBodyPartArmour.GetRatingValue(attackType, armorPenetration);
-				if (maxHealth - Damages[(int) damageType] < SubOrganDamageIncreasePoint)
-				{
-					organDamageRatingValue +=
-						1 - ((maxHealth - Damages[(int) damageType]) / SubOrganDamageIncreasePoint);
-					organDamageRatingValue = Math.Min(1, organDamageRatingValue);
-				}
-
-				var subDamage = damage * organDamageRatingValue;
-
-				//TODO: remove BodyPart component from organ
-				if (damageSplit)
-				{
-					foreach (var organ in OrganList)
-					{
-						var organBodyPart = organ.GetComponent<BodyPart>();
-						organBodyPart.TakeDamage(damagedBy, subDamage / OrganList.Count, attackType, damageType, damageSplit);
-					}
-				}
-				else
-				{
-					var organBodyPart = OrganList.PickRandom().GetComponent<BodyPart>(); //It's not like you can aim for Someone's liver can you
-					organBodyPart.TakeDamage(damagedBy, subDamage, attackType, damageType);
-				}
+				DamageOrgans(damage, attackType, damageType, organDamageSplit, armorPenetration);
 			}
 			if(damageType == DamageType.Brute) //Check damage type to avoid bugs where you can blow someone's head off with a shoe.
 			{
 				if (attackType == AttackType.Melee || attackType == AttackType.Laser || attackType == AttackType.Energy)
 				{
+					if (tramuticDamageType != TraumaticDamageTypes.NONE && DMMath.Prob(traumaDamageChance))
+					{
+						//TODO: move this to an utility, its hard to read! - picks a random enum from the ones already flagged
+						Random random = new Random();
+						TraumaticDamageTypes[] typeToSelectFrom = Enum.GetValues(typeof(TraumaticDamageTypes)).Cast<TraumaticDamageTypes>().Where(x => tramuticDamageType.HasFlag(x)).ToArray();
+						TraumaticDamageTypes selectedType = typeToSelectFrom[random.Next(1, typeToSelectFrom.Length)];
+						ApplyTraumaDamage(selectedType);
+					}
 					CheckBodyPartIntigrity(damage);
 				}
 			}
+
 			if(attackType == AttackType.Bomb)
 			{
 				TakeBluntDamage();
-				if(damageToLimb >= DamageThreshold) DismemberBodyPartWithChance();
+				if (damageToLimb >= DamageThreshold)
+				{
+					DismemberBodyPartWithChance();
+				}
 			}
 
-			if(Severity <= DamageSeverity.LightModerate) return;
-			if (damageType == DamageType.Burn || attackType == AttackType.Fire
-			                                  || attackType == AttackType.Laser || attackType == AttackType.Energy)
+			if (damageType == DamageType.Burn || attackType == AttackType.Fire ||
+			    attackType == AttackType.Laser || attackType == AttackType.Energy)
 			{
-				ApplyTraumaDamage(TraumaticDamageTypes.BURN, true);
```

Comment:
why was this pass removed?

Reply:
it never did anything due to the `if(Severity <= DamageSeverity.LightModerate) return;` line that I also removed a couple lines before
so now the lightmoderate check will still exist and used, but inside ApplyTraumaDamage

Reply:
Everything seems to be good, tested as well and no issues creeping up.